### PR TITLE
Week 4 Assignments

### PR DESCRIPTION
**Pull request by anugrahpaul2000**
Assignments Done

### CODE REVIEW

File: Week-4/Assignment-1/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function getNumber(delay) {
+  return new Promise((resolve, reject) => {
```
Comment (subhadip-pesto) [inline Week-4/Assignment-1/index.js]: Create your custom promise polyfill

File: Week-4/Assignment-2/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+class Person
```
Comment (subhadip-pesto) [inline Week-4/Assignment-2/index.js]: in javascript use braces like this

class Person {

}

The reason is most of the developers use this way of putting the braces, even later when you will be introduced to the linters, that will also follow the same.

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

**Comment by anugrahpaul2000** [comment]
@subhadip-pesto Updated Assignments

**Comment by anugrahpaul2000** [comment]
Updated Assignment @subhadip-pesto

**Review by subhadip-pesto — APPROVED** [review]

**Pull request merged**